tutorials/advanced_features/Local addressability with DMM.ipynb

```diff
@@ -0,0 +1,333 @@
+{
```

Comment:
<p><code>bottom_detuning</code> is not rendering correctly in the doc, presumably because its in <strong>bold </strong> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/569/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>